Body/AAUHuman/BodyModels/GenericBodyModel/Helper.ClassTemplates.any

```diff
@@ -21,6 +21,21 @@
 
 #define REPLACE_ZEROS(x, replacement) ((x) + not(x)*(replacement))
 
+// Create a difference matrix of of size (N,N-1)
+// This can be used calcuate the difference between adjacent elements in a vector or matrix
+#define DIFFMAT(N) (arrcat(zeros(1,N-1), eye(N-1)')' - arrcat(eye(N-1)', zeros(1,N-1))')
+
+// Pairwise distances between observations in n-dimensional space
+#define PDIST(X) sum(vnorm(DIFFMAT(SizesOf(X)[0])*(X)))
+
+// Upper triangle of the array. Return a copy of an array elements below the diagnonal zeroed out 
+#define TRIU(IN) mult(IN, gteqfun(transpose(repmat(1,SizesOf(IN)[0] ,iarr(1,SizesOf(IN)[1]))), repmat(1,SizesOf(IN)[1],iarr(1,SizesOf(IN)[0]))))
+
+// Lower triangle of the array. Return a copy of an array elements above the diagnonal zeroed out 
+#define TRIL(IN) mult(IN, lteqfun(transpose(repmat(1,SizesOf(IN)[0] ,iarr(1,SizesOf(IN)[1]))), repmat(1,SizesOf(IN)[1],iarr(1,SizesOf(IN)[0]))))
```

Comment:
```suggestion
// Create a difference matrix of of size (N,N-1)
// This can be used calcuate the difference between adjacent elements in a vector or matrix.
#define DIFFMAT(N) (arrcat(zeros(1,N-1), eye(N-1)')' - arrcat(eye(N-1)', zeros(1,N-1))')

// Pairwise distances between observations in n-dimensional space.
#define PDIST(X) sum(vnorm(DIFFMAT(SizesOf(X)[0])*(X)))

// Upper triangle of the array. Return a copy of an array with elements below the diagonal zeroed out 
#define TRIU(IN) mult(IN, gteqfun(transpose(repmat(1,SizesOf(IN)[0] ,iarr(1,SizesOf(IN)[1]))), repmat(1,SizesOf(IN)[1],iarr(1,SizesOf(IN)[0]))))

// Lower triangle of the array. Return a copy of an array with elements above the diagonal zeroed out 
#define TRIL(IN) mult(IN, lteqfun(transpose(repmat(1,SizesOf(IN)[0] ,iarr(1,SizesOf(IN)[1]))), repmat(1,SizesOf(IN)[1],iarr(1,SizesOf(IN)[0]))))
```